build/Windows/XPlatCppWindows.vcxproj

```diff
@@ -115,7 +115,6 @@
     <ClInclude Include="$(SdkSourceDir)\include\playfab\PlayFabAuthenticationContext.h" />
     <ClInclude Include="$(SdkSourceDir)\include\playfab\PlayFabApiSettings.h" />
     <ClInclude Include="$(SdkSourceDir)\include\playfab\PlayFabBaseModel.h" />
-
```

Comment:
These spaces are an artifact of the auto generation, and it's one of the ones we've just accepted.
It is fixable, but the template would become a lot less readable, so we opted to keep the spaces.

Reply:
(really, what will happen is that you just won't change this corresponding template file, and this change will be undone)

Reply:
[]()![20221124_202305.jpg](https://user-images.githubusercontent.com/100805074/204068574-32c4b7d5-1e07-4369-a735-af5f349b3a53.jpg)